ansible/configs/sap-integration/post_software.yml

```diff
@@ -0,0 +1,371 @@
+    when: test_deploy_results | default( False ) | bool
+    block:
+    - name: Set Ansible Python interpreter to k8s virtualenv
+      set_fact:
+        ansible_python_interpreter: /opt/virtualenvs/k8s/bin/python
+
+    - name: Get metadata.json
+      stat:
+        path: /home/{{ ansible_user }}/{{ cluster_name }}/metadata.json
+      register: r_metadata
+
+    - name: Get the infra ID
+      shell: jq -r .infraID /home/{{ ansible_user }}/{{ cluster_name }}/metadata.json
+      register: r_infra_id
+      when: r_metadata.stat.exists
+
+    - name: create fio testing project
+      k8s:
+        name: fio-test
+        api_version: v1
+        kind: Namespace
+        state: present
+
+    - name: give default sa privileged scc
+      shell: oc adm policy add-scc-to-user privileged system:serviceaccount:fio-test:default
+
+    - name: run job pod with fio-etcd-osp container
+      k8s:
+        state: present
+        definition: "{{ lookup('template', './files/fio-test-job.yaml.j2') }}"
+      vars:
+        INFRA_ID: "{{ r_infra_id.stdout }}"
+
+    - name: Wait for job to finish (1h max)
+      k8s_facts:
+        api_version: batch/v1
+        kind: Job
+        name: fio-test
+        namespace: fio-test
+      register: r_fio_test_job
+      retries: 60
+      delay: 60
+      until: r_fio_test_job.resources[0].status.conditions | json_query(fio_query) | bool
+      vars:
+        fio_query: >-
+          [?type=='Complete'].status[] | [0]
+        INFRA_ID: "{{ r_infra_id.stdout }}"
+
+    - name: Remove job
+      k8s:
+        name: fio-test
+        kind: Job
+        api_version: batch/v1
+        namespace: fio-test
+        state: absent
+
+    - name: Remove fio testing project
+      k8s:
+        name: fio-test
+        kind: Namespace
+        api_version: v1
+        state: absent
+
+- name: Deploy custom workloads in OCP
+  hosts: bastions
+  run_once: yes
+  gather_facts: false
+  become: false
+  environment:
+    KUBECONFIG: /home/{{ ansible_user }}/{{ cluster_name }}/auth/kubeconfig
+  tasks:
+    - name: Add additional tasks for specific workloads
+      include_tasks: custom_workloads.yml
+
+# This block may apply in the future for other demos where SAPGUI
+# could be required for further development
+# - name: Deploy SAP GUI in the bastion Host
+#   hosts: bastions
+#   run_once: yes
+#   gather_facts: false
+#   become: true
+#   tasks:
+#     - name: Add required packages for SAP GUI
+#       yum:
+#         name:
+#           - xauth
+#           - libXtst
+#           - /nfs/SAPGUI/jdk-8u251-linux-x64.rpm
+
+#     - name: Install SAP GUI
+#       command: java -jar /nfs/SAPGUI/PlatinGUI750Linux_1.JAR install -G
+
+- name: Restart HANA DB
+  hosts: hanas
+  gather_facts: false
+  become: true
+  tasks:
+    - name: Ensure HANA instance is stopped in the secondary node
+      shell: |
+          /usr/sap/{{ sap_hana_deployment_hana_sid | upper }}/HDB{{ sap_hana_deployment_hana_instance_number }}/HDB stop
```

Comment:
Where are these vars defined? I don't see them in default_vars

Reply:
These are stored as part of the secrets variable file

Reply:
Added those already in the sample_vars